images/ose-cli-artifacts.yml

```diff
@@ -30,4 +30,6 @@ owners:
 - aos-master@redhat.com
 konflux:
   vm_override:
-    aarch64: linux-d160/arm64
+    x86_64: linux-d160-m2xlarge/amd64
+    aarch64: linux-d160-m2xlarge/arm64
+    s390x: linux-large/s390x
```

Comment:
```suggestion
    x86_64: linux-d160-c4xlarge/amd64
    aarch64: linux-d160-c4xlarge/arm64
    s390x: linux-largecpu/s390x
```

Reply:
Should be usable once https://github.com/redhat-appstudio/infra-deployments/pull/5530 merges